tests/performance/log-results.js

```diff
@@ -108,6 +108,7 @@ const req = https.request( options, ( res ) => {
 
 req.on( 'error', ( error ) => {
 	console.error( error );
+	process.exit( 1 );
```

Comment:
Note to reviewers: This is not fixing any bug, but it should ensure that in the future any potential failures like this won't go unnoticed, like they did now for several months.